.github/workflows/release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           cache: "npm"
       - run: npm ci
       - run: npm run build documentation -- --base-href=/charts-on-fhir/
-      - run: cp dist/apps/documentation/index.html dist/apps/documentation/404.html # Needed for Angular routing
+      - run: cp dist/apps/documentation/browser/index.html dist/apps/documentation/404.html # Needed for Angular routing
```

Comment:
Documentation site is not working: https://elimuinformatics.github.io/charts-on-fhir/
You also need to add /browser to the cp destination path and the upload-pages-artifact path